composer.json

```diff
@@ -36,5 +36,11 @@
 		"psr-0": {
 			"SimplePie": "library"
 		}
-	}
+    },
+    "config": {
+        "bin-dir": "bin"
```

Comment:
This is a big change. We shouldn't do it unless @mblaney (the maintainer) is OK with it, and any dependent scripts have been updated.

Reply:
All right. I can remove this and use the previous approach.